articles/reporting/signals-analytics.md

```diff
@@ -0,0 +1,43 @@
+
+## Summary Report
+
+The default reporting view is the Summary report, as seen below. It provides you the ability to select from all metrics SparkPost tracks, adjust reporting date ranges, and filter by campaign, sending domain, recipient domain, and more. [Learn more](./summary) about the Summary report.
+
+![The summary report view in SparkPost](media/signals-analytics/summary-report.png)
+
+## Bounce, Rejections, Accepted, Delayed Reports
+
+Bounce, Rejections, Accepted, and Delayed reports in SparkPost provide the same date and filtering tools as the Summary report, with additional data and visualizations. Each report focuses on a different type of event and provides a breakdown into the different categories in the event, along with messages which provide information about why the event happened. For example, on the Bounce report, you'll see what reason mailbox providers are returning when they bounce your email.
+
+![](media/signals-analytics/bounce-report.png)
+
+
+## Health Score
+
+The Health Score is a predictive score to help you monitor your email health and identify problems before they become apparent through traditional metrics. Use it to discover areas of your sending that are low-performing and get recommendations on how to improve. [Learn more](./health-score) about the Health Score.
+
+![The overall health score for a SparkPost account](media/signals-analytics/health-score.png)
+
+## Spam Trap Monitoring
+
+Spam traps are email addresses that you should not send to, which are used by different organizations to find senders who don't follow good practices. If you send to too many of these addresses, your mail could end up being blocked, delayed or placed in the spam folder. Spam Trap Monitoring reports how many emails per day you are sending to spam traps so you can identify problems and improve your list management practices. [Learn more](./spam-trap-monitoring) about Spam Trap Monitoring.
+
+![A breakdown of the spam traps hits for a subaccount](media/signals-analytics/spam-traps.png)
+
+## Engagement Recency Cohorts
+
+You have a relationship with each person you send to. A powerful way to analyze that relationship is how recently they've engaged with your emails. The more recent they've engaged, the better your relationship. Each cohort is a group of recipients based on how recently they've engaged with your emails. For each day, Engagement Recency Cohorts display how much of your mail you sent to each group, so you know if your recipients are engaged with your email and you have a good relationship or if you need to remove disengaged folks and send more engaging content. [Learn more](./engagement-recency) about Engagement Recency Cohorts.
```

Comment:
Is a word missing from: "Each cohort is a group of recipients based on how recently they've engaged with your emails. ?
Move the "For each day" to after "group"? Also that sentence is a lot